explorer/ui/app/components/details/From.tsx

```diff
@@ -15,16 +16,22 @@ export const From = () => {
       {/* Offset */}
       <div className="flex mt-5 pb-2 border-b-2 border-gray-500 border-solid">
         <p className="w-48">Source Address</p>
-        <Link to="/" className="underline text-indigo-400">
+        <Link to="/" className="underline text-indigo-400 hidden lg:block">
           0x109f40f806567158aaad05e43afe240cf394608cacd0016466dfb24dce2927d4
         </Link>
+        <Link to="/" className="underline text-indigo-400 block lg:hidden">
+          {hashShortener('0x109f40f806567158aaad05e43afe240cf394608cacd0016466dfb24dce2927d4')}
+        </Link>
       </div>
       {/* Offset */}
       <div className="flex mt-5 pb-2 border-b-2 border-gray-500 border-solid">
         <p className="w-48">Tx Hash</p>
-        <Link to="/" className="underline text-indigo-400">
+        <Link to="/" className="underline text-indigo-400 hidden lg:block">
           0x109f40f806567158aaad05e43afe240cf394608cacd0016466dfb24dce2927d4
         </Link>
+        <Link to="/" className="underline text-indigo-400 block lg:hidden">
+          {hashShortener('0x109f40f806567158aaad05e43afe240cf394608cacd0016466dfb24dce2927d4')}
```

Comment:
why is this hash hard-coded?

Reply:
this is still old code this will be fixed in the next PR